src/status_im/navigation/core.cljs

```diff
@@ -25,7 +25,7 @@
    (fn []
      (navigation/set-default-options options/default-options)
      (reset! state/modals [])
-     (reset! state/dissmissing false)
+     (reset! state/dissmissing-count 0)
```

Comment:
Storing the dismiss state with true/false only works if we close one modal at a time. If two modals are closed simultaneously and another is opened, the state/modals behave like this: the first dismissed modal is removed, then the newly opened modal is added and immediately removed.